modules/las/package.json

```diff
@@ -35,8 +35,8 @@
     "README.md"
   ],
   "scripts": {
-    "pre-build": "npm run build-worker",
-    "pre-build-disabled": "npm run build-worker && npm run build-bundle && npm run build-bundle -- --env.dev",
+    "pre-build": "npm run build-worker && npm run build-bundle",
+    "pre-build-disabled": "npm run copy-libs && npm run build-worker && npm run build-worker-dev && npm run build-bundle && npm run build-bundle -- --env.dev",
```

Comment:
I don't see `copy-libs` script?

Reply:
> I don't see copy-libs script?

Well spotted. Yes this script is disabled, we still don't load scripts dynamically for the las-loader. That will be another PR